spec/controllers/api/v1/admin/tags_controller_spec.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Api::V1::Admin::TagsController do
+  render_views
+
+  let(:role)   { UserRole.find_by(name: 'Admin') }
+  let(:user)   { Fabricate(:user, role: role) }
+  let(:scopes)  { 'admin:read admin:write' }
```

Comment:
Should we have a test for access these APIs without the correct scope?

Reply:
I think I can do that.